static/src/javascripts/projects/commercial/modules/ipsos-mori.ts

```diff
@@ -24,22 +26,33 @@ const loadIpsosScript = () => {
  * documentation on DCR: [link](https://github.com/guardian/dotcom-rendering/blob/150fc2d81e6a66d9c3336185e874fc8cd0288546/dotcom-rendering/docs/architecture/3rd%20party%20technical%20review/002-ipsos-mori.md)
  * @returns Promise
  */
-export const init = (): Promise<void> =>
-	getLocale()
+export const init = (): Promise<void> => {
+	const forceIpsosMoriAustraliaTest = isInVariantSynchronous(
+		ipsosMoriAustralia,
+		'variant',
+	);
+
+	return getLocale()
 		.then((locale) => {
-			if (locale === 'GB') {
+			if (
+				locale === 'GB' ||
+				(locale === 'AU' && forceIpsosMoriAustraliaTest)
+			) {
 				return getInitialConsentState();
 			} else {
-				throw Error('Skipping ipsos outside of GB');
+				throw Error('Skipping ipsos process outside GB or AU');
 			}
 		})
 		.then((state) => {
-			if (getConsentFor('ipsos', state)) {
-				void loadIpsosScript();
+			if (state.aus) {
+				void loadIpsosScript('au');
```

Comment:
I'm guessing that in aus we don't need any additional consent to serve this script?

Reply:
Thanks Olly - good question. Yes, we've got the go ahead to serve this script in the 0% AB test without any further consent check for Australia.